buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -0,0 +1,238 @@
+            parameters(PARAMETERS)])
+*/
+
+timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
+    timestamps {
+        node(SETUP_LABEL) {
+            try {
+                def jobs = [:]
+                def offlineSlaves = []
+                def unavailableSlaves = []
+
+                def cleanDirs = ['NoEntryTest*.zip',
+                                 'auth*.login',
+                                 'tmp*',
+                                 'classes*',
+                                 'aci*',
+                                 'append*',
+                                 'mlib*',
+                                 'resource-*',
+                                 'openj9tr_resources*',
+                                 'testParentDir',
+                                 'jni-*',
+                                 'mauve',
+                                 'test*',
+                                 'blah-*.tmp',
+                                 'lines*.tmp',
+                                 'prefix*.json',
+                                 'sink*.tmp',
+                                 'source*.tmp',
+                                 'target*.tmp',
+                                 'sharedcacheapi',
+                                 'intermediateClassCreateTest']
+
+                for (aNode in jenkins.model.Jenkins.instance.getLabel(LABEL).getNodes()) {
+                    def nodeName = aNode.getDisplayName()
+
+                    if (aNode.toComputer().isOffline()) {
+                        // skip offline slave
+                        offlineSlaves.add(nodeName)
+                        continue
+                    }
+
+                    if (!aNode.toComputer().isAcceptingTasks()) {
+                        // node not ready, skip
+                        unavailableSlaves.add(nodeName)
+                        continue
+                    }
+
+                    def nodeLabels = []
+                    if (aNode.getLabelString()) {
+                        nodeLabels.addAll(aNode.getLabelString().tokenize(' '))
+                    }
+
+                    // cache job
+                    jobs["${nodeName}"] = {
+                        node("${nodeName}") {
+                            if (MODES.contains('cleanup')) {
+                                stage("${nodeName} - Cleanup Workspaces") {
+                                    def buildWorkspace = "${env.WORKSPACE}"
+                                    if (nodeLabels.contains('sw.os.windows')) {
+                                        // convert windows path to unix path
+                                        buildWorkspace = sh(script: "cygpath -u '${env.WORKSPACE}'", returnStdout: true).trim()
+                                    }
+
+                                    // Cleanup Test results
+                                    sh "cd /tmp && pwd && rm -rf ${cleanDirs.toString().replaceAll(',',' ')[1..-1]}"
```

Comment:
Is there an advantage of doing a `cd /tmp` rather than doing `dir('/tmp') { ... }` ?

Reply:
this can be cleaned